backend/editor/entries.py

```diff
@@ -82,59 +84,76 @@ async def create_node(self, label, entry, main_language_code):
         result = await get_current_transaction().run(" ".join(query), params)
         return (await result.data())[0]["n.id"]
 
-    async def parse_taxonomy(self, filename):
+    def parse_taxonomy(self, uploadfile=None):
         """
         Helper function to call the Open Food Facts Python Taxonomy Parser
         """
-        # Close current transaction to use the session variable in parser
-        await get_current_transaction().commit()
-
-        with SyncTransactionCtx() as session:
-            # Create parser object and pass current session to it
-            parser_object = parser.Parser(session)
-            try:
-                # Parse taxonomy with given file name and branch name
-                parser_object(filename, self.branch_name, self.taxonomy_name)
-                return True
-            except Exception as e:
-                raise TaxonomyParsingError() from e
+        if uploadfile is None:  # taxonomy is imported
+            base_url = (
+                "https://raw.githubusercontent.com/openfoodfacts/openfoodfacts-server"
+                "/main/taxonomies/"
+            )
```

Comment:
you must use the REPO_URI env variable for that
```suggestion
            default_url = "https://raw.githubusercontent.com/openfoodfacts/openfoodfacts-server"
            base_url = os.environ.get("REPO_URI", default_url) + "/main/taxonomies/"
```